lib/happo/snapshot_comparer.rb

```diff
@@ -1,19 +1,24 @@
 require 'oily_png'
 require 'diff-lcs'
-require_relative 'snapshot_comparison_image/base'
-require_relative 'snapshot_comparison_image/gutter'
-require_relative 'snapshot_comparison_image/before'
-require_relative 'snapshot_comparison_image/overlayed'
-require_relative 'snapshot_comparison_image/after'
 
 module Happo
   # This class is responsible for comparing two Snapshots and generating a diff.
   class SnapshotComparer
+    include ChunkyPNG::Color
+    BASE_OPACITY    = 0.1
+    BASE_ALPHA      = (255 * BASE_OPACITY).round
+    BASE_DIFF_ALPHA = BASE_ALPHA * 2
+    MAGENTA = ChunkyPNG::Color.from_hex '#b33682'
+    WHITE_OVERLAY = ChunkyPNG::Color.fade(WHITE, 1 - BASE_ALPHA)
+
     # @param png_before [ChunkyPNG::Image]
     # @param png_after  [ChunkyPNG::Image]
     def initialize(png_before, png_after)
       @png_after  = png_after
       @png_before = png_before
+
+      @diff_pixels  = {}
+      @faded_pixels = {}
```

Comment:
The name of this variable is confusing. The fact that these pixels are faded is an implementation detail of how we choose to represent them. I think it would be better to use a more semantic name, like we do for `diff_pixels`. How about `same_pixels` or `non_diff_pixels`?